src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -4686,3 +4686,239 @@ def test_ps_s3_notification_push_kafka_security_ssl_sasl_scram():
 def test_ps_s3_notification_push_kafka_security_sasl_scram():
     kafka_security('SASL_PLAINTEXT', mechanism='SCRAM-SHA-256')
 
+
+@attr('data_path_v2_test')
```

Comment:
since these tests cannit run in teuthology (for now), we should prevent that, by adding `!data_path_v2_test` to the list here: https://github.com/ceph/ceph/blob/main/qa/tasks/notification_tests.py#L223